### PR TITLE
Dec 22 2021

### PR DESCRIPTION
**Pull request by Rahul4321Kumar**
Added package integration of dj-rest-auth
1. Install dj-rest-auth
2. Integrate app in installed app
3. Perform makemigrations and migrate to create table in DB
4. Add urls of dj-rest-auth  in project urls.py
5. Install django-allauth and integrate in installed app
6. Add urls of registration in project.urls

### CODE REVIEW

File: EMS/ems/settings.py [hunk 1]
```diff
@@ -40,6 +40,13 @@
 
      #integrated apps
     'rest_framework',
+    'rest_framework.authtoken',
+    'dj_rest_auth',
+    'django.contrib.sites',
+    'allauth',
+    'allauth.account',
+    'allauth.socialaccount',
+    'dj_rest_auth.registration',
```
Comment (ksilawat-deqode) [inline EMS/ems/settings.py]: We can do this like
```suggestion
    'django.contrib.sites',

    #integrated apps
    'allauth',
    'allauth.account',
    'allauth.socialaccount',
    'dj_rest_auth',
    'dj_rest_auth.registration',
    'rest_framework',
    'rest_framework.authtoken',
```

File: EMS/ems/settings.py [hunk 2]
```diff
@@ -131,4 +138,7 @@
 
 AUTHENTICATION_BACKENDS = (
     ('django.contrib.auth.backends.ModelBackend'),
-)
\ No newline at end of file
+)
+
+#the database ID of the Site object associated with that particular settings file.
+SITE_ID = 1
```
Comment (ksilawat-deqode) [inline EMS/ems/settings.py]: Still need to point out for formatting.

File: EMS/ems/urls.py [hunk 3]
```diff
@@ -14,8 +14,10 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('dj-rest-auth/', include('dj_rest_auth.urls')),
+    path('dj-rest-auth/registration/', include('dj_rest_auth.registration.urls'))
```
Comment (ksilawat-deqode) [inline EMS/ems/urls.py]:
```suggestion
    path('authn/', include('dj_rest_auth.urls')),
    path('authn/registration/', include('dj_rest_auth.registration.urls'))
```

**Review by ksilawat-deqode — CHANGES_REQUESTED** [review]
Focus on formatting!

File: EMS/ems/settings.py (hunk 1 above)
Comment (Rahul4321Kumar) [reply EMS/ems/settings.py]: done

File: EMS/ems/urls.py (hunk 3 above)
Comment (Rahul4321Kumar) [reply EMS/ems/urls.py]: done

**Review by ksilawat-deqode — APPROVED** [review]
Looks good now!

**Pull request merged**